modules/third-party/org-apache-axis/src/main/java/org/apache/axis/description/JavaServiceDesc.java

```diff
@@ -811,7 +811,14 @@ public int compare(Method m1, Method m2) {
             }
             return (Method[])methodsList.toArray(new Method[]{}); 
         } else {
-            return implClass.getDeclaredMethods();
+            Method[] methods = implClass.getDeclaredMethods();
+            Arrays.sort(methods, new Comparator<Method>() {
```

Comment:
Why we only sort in the else branch? What about the if branch?
And see https://github.com/shuyangzhou/liferay-portal/pull/8616/commits/e68044f00dfa6b4a6b45d21776798d4d50b04b1f#diff-158c64206a1204e027521642a8ba1ee1R645
We already have a sorting there, if you want to enforce sorting to all calls of "getMethods()" then make sure to remove the outside additional sorting.